src/kbmod/search/image_kernels.cu

```diff
@@ -8,293 +8,80 @@
 #ifndef IMAGE_KERNELS_CU_
 #define IMAGE_KERNELS_CU_
 
-#define MAX_STAMP_IMAGES 200
-
 #include <assert.h>
 #include "common.h"
 #include "cuda_errors.h"
 #include <stdio.h>
 #include <float.h>
 
-#include "ImageStack.h"
-
 namespace search {
 
 /*
  * Device kernel that convolves the provided image with the psf
  */
-__global__ void convolvePSF(int width, int height, float *sourceImage, float *resultImage, float *psf,
-                            int psfRad, int psfDim, float psfSum, float maskFlag) {
+__global__ void ConvolvePSF(int width, int height, float *source_img, float *result_img, float *psf,
```

Comment:
isn't this a function, but we've named it capitalized camel case?

Reply:
Fixed.